tensorflow/python/ops/lookup_ops.py

```diff
@@ -419,6 +419,7 @@ def initialize(self, table):
     return init_op
 
 
+@tf_export("lookup.TextFileIndex")
 class TextFileIndex(object):
```

Comment:
Lets add some documentation here describing what these constants mean and how can one use them. I think they are described in the file elsewhere.

Reply:
The documentation helps a lot, but I still don't know which endpoints in the TF API accept a TextFileIndex. Mentioning it here is helpful